src/utils/network-monitor.ts

```diff
@@ -980,7 +980,7 @@ export class NetworkMonitor {
         this.structuredLog(job.network, `Block processing complete ✅`, job.block)
       }
 
-      this.updateLastProcessedBlock(job)
+      // this.updateLastProcessedBlock(job)
```

Comment:
@alexanderattar lets make sure we take care of this bug soon.

Reply:
what bug is this referring to here?